config/environments/development.rb

```diff
@@ -65,6 +65,8 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.community_api_host = 'https://community-api.dev.probation.service.justice.gov.uk'
```

Comment:
Any reason why you've hardcoded these URLs here?

We haven't done that for any of the other APIs we use. Is there any reason for this to be different?